docker-compose.yml

```diff
@@ -35,7 +34,7 @@ services:
   # Newer version of this image might run into this issue:
   # @link https://github.com/elgalu/docker-selenium/issues/20
   selenium:
-    image: selenium/standalone-chrome-debug:3.141.59-oxygen
+    image: registry.fpfis.eu/fpfis/selenium:standalone-chrome-debug-3.141.59
```

Comment:
Why not like in media_avportal?
![image](https://user-images.githubusercontent.com/22004498/99546187-780f7e00-29b6-11eb-8265-c2fb45a81e35.png)